main.js

```diff
@@ -1,132 +1,121 @@
-                        for (let j = 0; j < obj.collector.length; j++) {
-                            if (obj.collector[j].id == collectorToDeleteid[i]) {
-                                obj.collector.splice(j, 1);
-                            }
-                        }
-                    }
-                    for (let i = 0; i < ticketToDeleteid.length; i++) {
-                        for (let j = 0; j < obj.tickets.length; j++) {
-                            if (obj.tickets[j].id == ticketToDeleteid[i]) {
-                                obj.tickets.splice(j, 1);
-                            }
-                        }
-                    }
-                    // Write the new collectors.json
-                    json = JSON.stringify(obj);
-                    fs.writeFile('./collectors.json', json, 'utf8', function(err) {
-                        if (err) throw err;
-                        console.log("collectors.json Updated")
-                    });
-                }
-            }
-            , 1000);
+          }
+          // Write the new collectors.json
+          const json = JSON.stringify(obj);
+          fs.writeFile('./collectors.json', json, 'utf8', (err) => {
+            if (err) throw err;
+            console.log('collectors.json Updated');
+          });
         }
-    });
-})
+      }, 1000);
+    }
+  });
+});
 
-fs.readdir("./events/", (err, files) => {
+fs.readdir('./events/', (err, files) => {
   if (err) return console.error(err);
-  files.forEach(file => {
-      const event = require(`./events/${file}`);
-      let eventName = file.split(".")[0];
-      console.log(`Loading event ${eventName}`);
-      client.on(eventName, event.bind(null, client));
-    });
+  files.forEach((file) => {
+    const event = require(`./events/${file}`);
+    let eventName = file.split('.')[0];
+    console.log(`Loading event ${eventName}`);
+    client.on(eventName, event.bind(null, client));
+  });
 });
 
 client.commands = new Collection();
 
-fs.readdir("./commands/", (err, files) => {
-    if (err) return console.error(err);
-    files.forEach(file => {
-        if (!file.endsWith(".js")) return;
-        let props = require(`./commands/${file}`);
-        let commandName = file.split(".")[0];
-        console.log(`Loaded ${commandName}`);
-        client.commands.set(commandName, props);
-    });
+fs.readdir('./commands/', (err, files) => {
+  if (err) return console.error(err);
+  files.forEach((file) => {
+    if (!file.endsWith('.js')) return;
+    let props = require(`./commands/${file}`);
+    let commandName = file.split('.')[0];
+    console.log(`Loaded ${commandName}`);
+    client.commands.set(commandName, props);
+  });
 });
 
 client.login(token.token);
+
+const checkMessage = async (id, channelid) => {
+  const channel = await client.channels.cache.get(channelid);
+  if (!channel) return;
+  const channelMessage = await channel?.messages.fetch(id);
```

Comment:
Might want to add a .catch to prevent the application from crashing because if there is an error it is most likely because the message doesn't exist.

Reply:
if message doesn't exist the `channelMessage` will just be undefined. No error here.

Reply:
If the message doesn't exist the channelMessage will crash the application because Discord returns a 404 error code with `DiscordAPIError: Unknown Message`  so I suggest adding a `.catch` there.